dev/lint-python

```diff
@@ -23,6 +23,7 @@ PATHS_TO_CHECK="./python/pyspark/ ./examples/src/main/python/ ./dev/sparktestsup
 # TODO: fix pep8 errors with the rest of the Python scripts under dev
```

Comment:
Hi, @HyukjinKwon .
Is this the last PEP8 fix? If then, could you remove the above `TODO`?
If not, what about fixing all error in this PR?